pyiron_base/jobs/job/extension/server/generic.py

```diff
@@ -476,6 +477,8 @@ def executor(self, exe: Union[Executor, None]):
         """
         if isinstance(exe, Executor):
             self.run_mode.executor = True
+        elif static_isinstance(exe, "flux.job.executor.FluxExecutor"):
+            self.run_mode.executor = True
```

Comment:
This part is no longer necessary once https://github.com/flux-framework/flux-core/pull/5323 is included in the next flux version.